flight/PiOS/Common/pios_hal.c

```diff
@@ -48,6 +47,18 @@
 #include <openlrs.h>
 #endif
 
+#ifdef PIOS_INCLUDE_USART
+#include <pios_usart_priv.h>
```

Comment:
Just define anonymous structs for this stuff in public headers :-P. We should seriously look at taking the hit and doing that throughout pios so we can avoid including private headers everywhere like this. :wink: